src/collectives/device/msccl_interpreter.h

```diff
@@ -178,8 +179,7 @@ namespace {
         }
         if (msccltran->has_dependence && tid == nthreads-1){
 	        if (needsFence) __threadfence();
-          uint64_t curFlag = COMPUTE_FLAG(workIndex, iter, step);
-          mscclFlags[bid].flag = curFlag;
+          mscclFlags[bid].flag = static_cast<uint64_t>(COMPUTE_FLAG(workIndex, iter, step));
```

Comment:
one last thing, can you please make it `(uint64_t) COMPUTE_FLAG`. Thanks a lot!

Reply:
done!